CPAC/utils/interfaces/netcorr.py

```diff
@@ -0,0 +1,217 @@
+        "refuse to calculate if any ROI contains >10 percent "
+        "of voxels with null times series (i.e., each point is "
+        "0), as of April, 2017.  This is because it seems most "
+        "likely that hidden badness is responsible. However, "
+        "if the user still wants to carry on the calculation "
+        "anyways, then this option will allow one to push on "
+        "through.  However, if any ROI *only* has null time "
+        "series, then the program will not calculate and the "
+        "user will really, really, really need to address their masking",
+        argstr="-push_thru_many_zeros",
+    )
+    ignore_LT = Bool(
+        desc="switch to ignore any label table labels in the "
+        "'-in_rois' file, if there are any labels attached",
+        argstr="-ignore_LT",
+    )
+    out_file = File(
+        desc="output file name part",
+        name_template="%s_netcorr",
+        argstr="-prefix %s",
+        position=1,
+        name_source="in_file",
+    )
+
+
+class NetCorrOutputSpec(AFNICommandOutputSpec):
+    out_corr_matrix = File(
+        desc="output correlation matrix between ROIs written to a text "
+             "file with .netcc suffix"
+    )
+    out_corr_maps = List(
+        File(), desc="output correlation maps in Pearson and/or Z-scores"
+    )
+
+
+class NetCorr(AFNICommand):
+    """Calculate correlation matrix of a set of ROIs (using mean time series of
+    each). Several networks may be analyzed simultaneously, one per brick.
+
+    For complete details, see the `3dNetCorr Documentation
+    <https://afni.nimh.nih.gov/pub/dist/doc/program_help/3dNetCorr.html>`_.
+
+    Examples
+    --------
+    >>> from nipype.interfaces import afni
+    >>> ncorr = afni.NetCorr()
+    >>> ncorr.inputs.in_file = 'functional.nii'
+    >>> ncorr.inputs.mask = 'mask.nii'
+    >>> ncorr.inputs.in_rois = 'maps.nii'
+    >>> ncorr.inputs.ts_wb_corr = True
+    >>> ncorr.inputs.ts_wb_Z = True
+    >>> ncorr.inputs.fish_z = True
+    >>> ncorr.inputs.out_file = 'sub0.tp1.ncorr'
+    >>> ncorr.cmdline
+    '3dNetCorr -prefix sub0.tp1.ncorr -fish_z -inset functional.nii -in_rois maps.nii -mask mask.nii -ts_wb_Z -ts_wb_corr'
+    >>> res = ncorr.run()  # doctest: +SKIP
+
+    """  # noqa E501  # pylint: disable=line-too-long
+    _cmd = "3dNetCorr"
+    input_spec = NetCorrInputSpec
+    output_spec = NetCorrOutputSpec
+
+    def _list_outputs(self):
+        outputs = self.output_spec().get()
+
+        if not isdefined(self.inputs.out_file):
+            prefix = self._gen_fname(self.inputs.in_file, suffix="_netcorr")
+        else:
+            prefix = self.inputs.out_file
+
+        # All outputs should be in the same directory as the prefix
+        odir = os.path.dirname(os.path.abspath(prefix))
+        outputs["out_corr_matrix"] = glob.glob(
+            os.path.join(odir, "*.netcc"))[0]
+
+        if (
+            isdefined(self.inputs.ts_wb_corr) or
+            isdefined(self.inputs.ts_wb_Z)
```

Comment:
I changed this line because it was referencing a non-existent `self.inputs.ts_Z_corr`